src/renderer/store/modules/profiles.js

```diff
@@ -214,6 +232,22 @@ const mutations = {
     state.profileList.sort(profileSort)
   },
 
+  addChannelToProfiles(state, { channel, profileIds }) {
+    for (const id of profileIds) {
+      state.profileList.find(profile => profile._id === id).subscriptions.push(channel)
+    }
+  },
+
+  removeChannelFromProfiles(state, { channelId, profileIds }) {
+    for (const id of profileIds) {
+      const profile = state.profileList.find(profile => profile._id === id)
+
+      // use filter instead of splice in case the subscription appears multiple times
+      // https://github.com/FreeTubeApp/FreeTube/pull/3468#discussion_r1179290877
```

Comment:
**question (non-blocking, minor):** Doesn't have to be considered for this PR, but wouldn't a user know that they're subscribed to a channel multiple times? That + the amount of time since this bug has been fixed (unsure exactly when) means maybe this is safe to make a `splice` now.